From 0c278cf19c6fd24b70297100022fc5a3d6df169c Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Tue, 5 May 2015 16:11:36 +0800
Subject: [PATCH] Revert "usb: ftp: fix the bug of ftp ioctrl args problem with
 32bit userspace on 64bit platforms"

This reverts commit c04cdecff9f7283308b286e1bc1dbf0edfe4453b.
---
 include/uapi/linux/usb/f_mtp.h | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/include/uapi/linux/usb/f_mtp.h b/include/uapi/linux/usb/f_mtp.h
index 9a08c643ae20..503291855abd 100644
--- a/include/uapi/linux/usb/f_mtp.h
+++ b/include/uapi/linux/usb/f_mtp.h
@@ -38,22 +38,12 @@ struct mtp_file_range {
 	uint32_t	transaction_id;
 };
 
-/* support of 32bit userspace on 64bit platforms */
-#ifdef CONFIG_COMPAT
-struct mtp_event {
-	/* size of the event */
-	int32_t		length;
-	/* event data to send */
-	compat_uptr_t		data;
-};
-#else
 struct mtp_event {
 	/* size of the event */
 	size_t		length;
 	/* event data to send */
 	void		*data;
 };
-#endif
 
 /* Sends the specified file range to the host */
 #define MTP_SEND_FILE              _IOW('M', 0, struct mtp_file_range)
-- 
2.35.3

